eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Ledger.hs

```diff
@@ -158,7 +158,7 @@ instance
         ( \(TRC (LedgerEnv {ledgerPp}, _, _))
            (LedgerState utxoSt DPState {dpsDState, dpsPState}) ->
               obligation ledgerPp (rewards dpsDState) (_pParams dpsPState)
-                == _deposited utxoSt
+                == utxosDeposited utxoSt
```

Comment:
Just curious, is it `utxos` as in the plural of `utxo` or from `utxoState`  ?

Reply:
:+1:  It stands for `UTxO{S}tate` as in `State`, and not for plurality. :slightly_smiling_face: